### ISSUE TITLE
Operational status broken in v23.09

### ISSUE THREAD

**Issue by troglobit** [kernelkit/infix#151]
With the merge of #149 to `main` we broke `statd`, because it turns out we have implemented read-back of MTU in operational data, although configure support is not yet available.  So the change to disable the MTU YANG nodes cause the following error:

```
Oct  2 12:19:46 infix-00-00-00 statd: lydx_new_path: Failed building data tree, xpath /ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/mtu, libyang error 7: Not found node "mtu" in path.
Oct  2 12:19:46 infix-00-00-00 statd: ly_add_ip_mtu: Error, adding 'mtu' to data tree, libyang error 7
Oct  2 12:19:46 infix-00-00-00 statd: ly_add_ip_data: Error, adding ip MTU
Oct  2 12:19:46 infix-00-00-00 statd: ly_add_ip_addr: Error, adding ipv4 addr info for eth0
Oct  2 12:19:46 infix-00-00-00 statd: sr_ifaces_cb: Error, adding ip link info
Oct  2 12:19:46 infix-00-00-00 statd[2084]: libyang[0]: Not found node "mtu" in path. (path: Schema location "/ietf-interfaces:interfaces/interface/ietf-ip:ipv4".)
```

Since all operational data is returned, all operational functionality is broken as of v23.09.

----

**Original subject:** Unstable tests: `iface_status` and `iface_phys_address`

https://github.com/kernelkit/infix/actions/runs/6377786313/job/17307101738?pr=150#step:10:236

- iface_phys_address.py
- iface_status.py

### iface_phys_address.py

```
Starting test 0017-iface_phys_address.py
2023-10-02 09:11:26 # Starting (2023-10-02 09:11:23)
2023-10-02 09:11:26 # Probing dut1 on port d1a for IPv6LL mgmt address ...
2023-10-02 09:11:26 # Connecting to mgmt IP fe80::ff:fe00:0%d1a:830 ...
2023-10-02 09:11:26 # Target iface eth2 original mac None
2023-10-02 09:11:26 ok 1 - Initialize
2023-10-02 09:11:26 # Target iface eth2 current mac: None
2023-10-02 09:11:26 not ok 2 - Set custom MAC address
2023-10-02 09:11:26 # Exiting (2023-10-02 09:11:26)
2023-10-02 09:11:26 # Traceback (most recent call last):
2023-10-02 09:11:26 #   File "/home/runner/work/infix/infix/test/case/ietf_interfaces/iface_phys_address.py", line 30, in <module>
2023-10-02 09:11:26 #     assert mac == cmac
2023-10-02 09:11:26 #            ^^^^^^^^^^^
2023-10-02 09:11:26 # AssertionError
2023-10-02 09:11:26 #
```

### iface_status.py

```
Starting test 0018-iface_status.py
2023-10-02 09:11:30 # Starting (2023-10-02 09:11:27)
2023-10-02 09:11:30 # Probing dut1 on port d1a for IPv6LL mgmt address ...
2023-10-02 09:11:30 # Connecting to mgmt IP fe80::ff:fe00:0%d1a:830 ...
2023-10-02 09:11:30 ok 1 - Initialize
2023-10-02 09:11:30 # Failed to get interfaces' status from target <infamy.netconf.Device object at 0x7ff785395350>
2023-10-02 09:11:30 # Interfaces under test: ['lo', 'eth0']
2023-10-02 09:11:30 ok 2 - Verifying <lo> interface
2023-10-02 09:11:30 ok 3 - Verifying <if-index> for <lo> interface
2023-10-02 09:11:30 ok 4 - Verifying <oper-status> for <lo> interface
2023-10-02 09:11:30 not ok 5 - Verifying <eth0> interface
2023-10-02 09:11:30 # Exiting (2023-10-02 09:11:30)
2023-10-02 09:11:30 # Traceback (most recent call last):
2023-10-02 09:11:30 #   File "/home/runner/work/infix/infix/test/case/ietf_interfaces/iface_status.py", line 42, in <module>
2023-10-02 09:11:30 #     asser_iface_exists(target, interface)
2023-10-02 09:11:30 #   File "/home/runner/work/infix/infix/test/case/ietf_interfaces/iface_status.py", line 25, in asser_iface_exists
2023-10-02 09:11:30 #     assert iface.interface_exist(target, interface), f"Interface <{interface}> does not exist."
2023-10-02 09:11:30 # AssertionError: Interface <eth0> does not exist.
```

**Comment by troglobit** [comment]
Reproduced locally, investigating.

**Comment by troglobit** [comment]
Fixed in #152.
